modules/virt-virtual-gpus-config-overview.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: REFERENCE
+[id="configuration-overview_{context}"]
+= Configuration overview
+
+When configuring mediated devices, an administrator must complete the following tasks:
+
+* Create the mediated devices.
+* Expose the mediated devices to the cluster.
+
+The `HyperConverged` CR includes APIs that accomplish both tasks.
+
+.Creating mediated devices
+
+[source,yaml]
+----
+...
+spec:
+  mediatedDevicesConfiguration:
+    mediatedDevicesTypes: <1>
+    - <device_type>
+    nodeMediatedDeviceTypes: <2>
+    - mediatedDevicesTypes: <3>
+      - <device_type>
+      nodeSelector: <4>
+        <node_selector_key>: <node_selector_value>
+...
+----
+<1> Required: Configures global settings for the cluster.
+<2> Optional: Overrides the global configuration for a specific node or group of nodes. Must be used with the global `mediatedDevicesTypes` configuration.
+<3> Required if you use `nodeMediatedDeviceTypes`. Overrides the global `mediatedDevicesTypes` configuration for the specified nodes.
+<4> Required if you use `nodeMediatedDeviceTypes`. Must include a `key:value` pair.
+
+.Exposing mediated devices to the cluster
+
+[source,yaml]
+----
+...
+  permittedHostDevices:
+    mediatedDevices:
+    - mdevNameSelector: GRID T4-2Q <1>
+      resourceName: nvidia.com/GRID_T4-2Q <2>
+...
+----
+<1> Exposes the mediated devices that map to this value on the host.
++
+[NOTE]
+====
+You can see the mediated device types that your device supports by viewing the contents of `/sys/bus/pci/devices/<slot>:<bus>:<domain>.<function>/mdev_supported_types/<type>/name`, substituting the correct values for your system.
+
+For example, the name file for the `nvidia-231` type contains the selector string `GRID T4-2Q`. Using `GRID T4-2Q` as the `mdevNameSelector` value allows nodes to use the `nvidia-231` type.
+====
+<2> The `resourceName` should match that allocated on the node. Find the `resourceName` by using the following command:
++
+[source,terminal]
+----
+$ oc get $NODE -o json  | jq '.status.allocatable | with_entries(select(.key | startswith("nvidia.com/"))) | with_entries(select(.value != "0"))'
```

Comment:
I'd like to pause on merge and consider breaking up this command into multiple lines. According to the preview build at least, the code block in the callout is extending past the edge of the document:

https://53509--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.html#configuration-overview_virt-configuring-mediated-devices